### PR TITLE
rewrite of the "productivity tools" episode

### PR DESCRIPTION
**Pull request by bast**
- I found the previous version too dated and the examples too complex
- Still listing all tools I know but only showing one which can both check and format
- Mention AI tools with screenshots
- Simpler examples

### CODE REVIEW

File: content/productivity.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+
+## AI-assisted coding
+
+We can use AI as an assistant/apprentice:
+- Code completion
+- Write a test based on an implementation
+- Write an implementation based on a test
+
+Of we can use AI as a mentor:
```
Comment (samumantha) [inline content/productivity.md:160]:
```suggestion
Or we can use AI as a mentor:
```

File: content/productivity.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+- Explain a concept
+- Improve code
+- Show a better alternative
```
Comment (samumantha) [inline content/productivity.md:163]:
```suggestion
- Show a better alternative
```
Do you mean a different way of implementing the same thing?

**Review by samumantha — APPROVED** [review]
Two small things to check/approve, otherwise LGTM :)

File: content/productivity.md (hunk 2 above)
Comment (bast) [reply content/productivity.md:163]: Yes. I will rephrase. Thank you!

**Pull request merged**